common/src/main/java/org/candlepin/common/jackson/DynamicPropertyFilter.java

```diff
@@ -28,12 +32,27 @@
  */
 public class DynamicPropertyFilter extends CheckableBeanPropertyFilter {
 
+    private List<String> path = new ArrayList<String>(10);
+
     public boolean isSerializable(Object obj, JsonGenerator jsonGenerator,
         SerializerProvider serializerProvider, PropertyWriter writer) {
-        DynamicFilterData filterData =
-            ResteasyProviderFactory.getContextData(DynamicFilterData.class);
+
+        DynamicFilterData filterData = ResteasyProviderFactory.getContextData(DynamicFilterData.class);
+
         if (filterData != null) {
-            return !filterData.isAttributeExcluded(writer.getName(), obj);
+            this.path.clear();
```

Comment:
Saves some reallocation (assuming Java doesn't throw it on the stack) and initialization in a method that is run a lot.